### ISSUE TITLE
Workflow install via command line fails

### ISSUE THREAD

**Issue by boboldehampsink** [engram-design/Workflow#51]
### Description

Workflow install via command line fails. This is because `Service.php` events expect body params when there are none in console. Error:

```bash
web_1         |     > applying existing project config ... Exception: Calling unknown method: craft\console\Request::getBodyParam() (/app/user/vendor/yiisoft/yii2/base/Component.php:300)
web_1         | #0 /app/user/vendor/verbb/workflow/src/services/Service.php(24): yii\base\Component->__call('getBodyParam', Array)
web_1         | #1 [internal function]: verbb\workflow\services\Service->onBeforeSaveEntry(Object(craft\events\ModelEvent))
```

### Steps to reproduce

1. Install Workflow via cli

### Additional info

- Plugin version: 1.1.2
- Craft version: 3.1.15

**Comment by engram-design** [comment]
How're you installing Workflow? `./craft install/plugin workflow` I imagine?

I wouldn't have thought that `Entry::EVENT_BEFORE_SAVE` would be called during install - that makes very little sense to me! Are you doing anything else on the CLI?

**Comment by boboldehampsink** [comment]
I'm installing via Project Config. Before debugging all too much I guess it's easily fixed by wrapping it in a isConsole check right?

**Comment by engram-design** [comment]
Oh, easy fix for sure, just scratching my head about it. Anyway - who cares, project config sounds about right. Fixed in [1.1.3](https://github.com/verbb/workflow/releases/tag/1.1.3)
